tests/api2/test_023_kubernetes.py

```diff
@@ -82,7 +82,7 @@ def test_07_kubernetes_pods_stats(request):
                     break
 
             if timeout <= 0:
-                fail('Time to setup kubernetes exceeded 150 seconds')
+                pytest.fail('Time to setup kubernetes exceeded 150 seconds')
```

Comment:
`pytest.fail` should not be used in our code as it can lead to effects described in `middlewared.test.integration.utils.fail` comment

We should fail the entire dependency (`setup_kubernetes` in this case) to prevent k3s tests from running